go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -1722,8 +1719,8 @@ var _ = ginkgo.Describe("OVN cluster-manager EgressIP Operations", func() {
 					&egressipv1.EgressIPList{Items: []egressipv1.EgressIP{eIP}},
 				)
 
-				egressNode1 := setupNode(node1Name, []string{node1IPv6OVNManaged, node1IPv6NonOVNManaged}, map[string]string{"0:0:0:0:0:feff:c0a8:8e32": "bogus1", "0:0:0:0:0:feff:c0a8:8e1e": "bogus2"})
-				egressNode2 := setupNode(node2Name, []string{node2IPv6OVNManaged, node2IPv6NonOVNManaged}, map[string]string{"0:0:0:0:0:feff:c0a8:8e23": "bogus3"})
+				egressNode1 := setupNode(node1Name, []string{node1IPv6OVNManaged}, map[string]string{"0:0:0:0:0:feff:c0a8:8e32": "bogus1", "0:0:0:0:0:feff:c0a8:8e1e": "bogus2"})
```

Comment:
Nice! Could you please add a comment in the commit (or in the PR) on how you ran this specific test?

Reply:
What do you mean? 
I added either a focus to the describe or the individual ginkgo test.

Reply:
exactly that. I know it is obvious to us, but this is an opportunity to register how folks who are not familiar
with the test on running it. So this is all, and we can just leave it here (no further action needed :)):
```
cd ./go-controller

RACE="yes" PKGS="github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn" \
    hack/test-go.sh focus "OVN cluster-manager EgressIP Operations"
```

Reply:
Ahh nice! I was manually editing the go files and executed go test manually (with race enabled)! 
Thanks Flavio!